packages/fxa-settings/src/components/UnitRowRecoveryKey/index.tsx

```diff
@@ -75,17 +75,20 @@ export const UnitRowRecoveryKey = () => {
       </LinkExternal>
       {modalRevealed && (
         <VerifiedSessionGuard onDismiss={hideModal} onError={onError}>
-          {/* TODO customize confirm button text and style */}
           <Modal
             onDismiss={hideModal}
             onConfirm={deleteRecoveryKey}
+            // TODO - Uncomment these two
+            // lines once #6302 is merged
+            // confirmBtnClassName="cta-caution"
+            // confirmText="Remove"
             headerId="some-id"
             descId="some-desc"
```

Comment:
headerId/descId here should be something like `recovery-key-header` and `recovery-key-desc` but I know that's not part of your PR 🤷‍♀️

Reply:
I can do that.